packages/fxa-settings/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../../_dev/tsconfig.browser.json",
   "compilerOptions": {
+    "experimentalDecorators": true,
```

Comment:
I know it says experiemental, but we are using these all over the place, for example nest relies on decorators. Looking at the JS code that gets generated decorators are actually pretty straight forward, and I've used them in production systems before without issue.